tests/unit/config/comm/test_ipc.py

```
@@ -0,0 +1,199 @@
+    @pytest.fixture(autouse=True)
+    def _force_not_windows(self):
+        with patch("aiperf.config.comm.ipc.IS_WINDOWS", False):
+            yield
+
+    def test_returns_ipc_url_with_path_and_filename(self, tmp_path: Path) -> None:
+        address = _build_socket_address(tmp_path, "event_bus.ipc")
+        assert address == f"ipc://{tmp_path / 'event_bus.ipc'}"
+
+    def test_path_none_raises_value_error(self) -> None:
+        with pytest.raises(
+            ValueError, match="[Pp]ath is required for socket address derivation"
```

Ruff (0.15.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)